src/HTSPDemuxer.cpp

```diff
@@ -309,13 +309,8 @@ bool CHTSPDemuxer::IsRealTimeStream() const
    * we want the calculation to be consistent */
   CLockObject lock(m_mutex);
 
-  if (m_timeshiftStatus.shift == 0)
-    return true;
-
-  if (m_timeshiftStatus.start - m_timeshiftStatus.shift < 10)
-    return true;
-
-  return false;
+  /* Handle as real time when reading close to the EOF (10000000�s - 10s) */
+  return (m_timeshiftStatus.shift < 10000000);
```

Comment:
I'm not at all sure about this change. A stream that is 10 secs off real time is considered 'real time'? Really? What does this change fix?

@FernetMenta what's your take on this?